dask_ctl/cli.py

```diff
@@ -292,7 +292,7 @@ async def _list_discovery():
             )
         console.print(table)
 
-    loop.run_sync(_list_discovery)
+    run_sync(_list_discovery)
```

Comment:
the `_list_discovery` coroutine doesn't have any `awaits` so this `run_sync` isn't needed

Reply:
That's fair. Perhaps in this case the wrapping should go the other way so we implement it as a sync function and wrap it in a coroutine in `dask_ctl.asyncio`.